dsl/contracts/ValidatorSmartContractAllowList.sol

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+pragma solidity >=0.5.0;
+
+import "./ValidatorSmartContractInterface.sol";
+
+contract ValidatorSmartContractAllowList is ValidatorSmartContractInterface {
```

Comment:
Just a reminder: Are we keeping the simplified version?

Reply:
Yeah, it's easier to deal with in the tests :smile: